projects/adf-office-services-ext/src/lib/evaluators.ts

```diff
@@ -63,5 +63,11 @@ export function canOpenWithOffice(
     return false;
   }
 
-  return true;
+  if (file.entry.hasOwnProperty('allowableOperationsOnTarget')) {
+    return context.permissions.check(file, ['update'], {
```

Comment:
Can you please move the check to the https://github.com/Alfresco/alfresco-content-app/blob/development/src/app/services/node-permission.service.ts#L59, or fix the code there? Every rule has access to that service and might be better having this check there so that everyone can beneft.